### PR TITLE
Add Introduction, Product Overview and Budgie in edit section

### PR DESCRIPTION
**Pull request by jyxhazcake**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-for tech-savvy students who have trouble keeping track of their expenses.
+# Budget Tracker User Guide
 
 ## Product Overview
-
+Budget Tracker is a free, easy-to-use desktop app for managing finances, optimized for use via a Command Line Interface (CLI)
+for tech-savvy students who have trouble keeping track of their expenses. 
+You can use Budget Tracker to manage your budget, expenditures and loans lent out to other parties for the month, and 
+easily access and retrieve this information with a single command. 
+To aid you in spending better, Budget Tracker also displays useful data and statistics about your spending habits for the month.
```
Comment (ZenoNexus) [inline docs/UserGuide.md:8]: This could be changed to "To improve your spending habits, ..."

File: docs/UserGuide.md [hunk 2]
```diff
@@ -259,7 +276,10 @@ _________________________________________________________
 
 ### <a id="edit-expenditure"></a> `-e` : Edit an Expenditure 
 
-Edits the values of an expenditure, including its _description_, _amount_, _date of expenditure_ and _category_.
+> 😃 An expenditure record has many values, and editing everything is hassle for just one mistake.
```
Comment (ZenoNexus) [inline docs/UserGuide.md:279]: 1. not very clear what it means by "editing everything"?
2 *is a hassle

**Review by ZenoNexus — CHANGES_REQUESTED** [review]
minor grammar errors and unclear points

**Review by ZenoNexus — APPROVED** [review]
lgtm

**Review by LS-Young — APPROVED** [review]
lgtm

**Pull request merged**